### DISCUSSION TITLE
希望加入固态硬盘缓存插件实现缓存加速机械硬盘读写

### DISCUSSION THREAD

**Discussion by mawst** [istoreos/istoreos#1471] [Ideas]
目前发现内核并不支持bcache，希望软件中心上架固态缓存相关的插件，或者在新版本的内核中加入相关模块，以支持用户自己实现bcache。
PS：LVM缓存因为没有device-mapper，也无法实现，但LVM有点重量级了，感觉没必要上

**Comment by jjm2473** [comment]
lvm是支持的，现在从ventoy启动的时候，实际底层就是用lvm实现镜像文件到块设备的映射。bcache的话，添加模块倒是简单，只是配置对于一般用户也太难了。

**Reply by mawst** [reply]
能否出个教程，或者做个插件来实现lvm的webUI设置呢？个人在命令行是没成功。。。技术比较渣

**Comment by pizh-github** [comment]
同求，希望可以有一个这样的插件，如果istoreNAS想要走得远，这种功能是必不可少的。
